### ISSUE TITLE
Inconsistent indentation suggestions for Python

### ISSUE THREAD

**Issue by terrencepreilly** [google/styleguide#417]
The Python style guide for docstrings gives inconsistent advice for indentation.  It initially says,

> Sections should be indented two spaces, except for the heading

Yet all of the examples feature four spaces of indentation.  Further, under the *Args* section, it says,

> If the description is too long to fit on a single 80-character line, use a hanging indent of 2 or 4 spaces (be consistent with the rest of the file).

Probably the first quote should read "Sections should be indented two or four spaces (be consistent with the rest of the file), except for the heading."

**Comment by gpshead** [comment]
Thanks, internally we recommend two for this as it matches our internal coding convention. But I like your suggestion of just saying use either within the docstrings so long as it is consistent within the file. There is no strong reason to _require_ four for docstring section indentation regardless of PEP-8 code four space indentation.

**Comment by hevrard** [comment]
@gpshead as you said internally we recommend *two* spaces for code indentation. But this style guide recommands *four*. I think it should be changed to two?
See http://google.github.io/styleguide/pyguide.html#34-indentation

**Comment by gpshead** [comment]
Nope.  The world at large should not be infected with Google's internal style quirks.  This guide keeps in line with PEP-8 and what the entire world's Python editor configs use: four space indentation.

**Comment by hevrard** [comment]
Ok that makes sense. Thanks for the clarification!
